spec/Vehicle/Vehicle.vspec

```diff
@@ -184,16 +184,50 @@ Speed:
   description: Vehicle speed.
 
 TravelledDistance:
+  deprecation: V3.1 moved to Vehicle.TraveledDistance
   datatype: float
   type: sensor
   unit: km
-  description: Odometer reading, total distance travelled during the lifetime of the vehicle.
+  description: Odometer reading, total distance traveled during the lifetime of the vehicle.
 
-TripMeterReading:
+TraveledDistance:
+  datatype: float
+  type: sensor
+  unit: km
+  description: Odometer reading, total distance traveled during the lifetime of the vehicle.
+
+TraveledDistanceSinceStart:
   datatype: float
   type: sensor
   unit: km
-  description: Current trip meter reading.
+  description: Distance traveled since start of current trip.
+  comment: A new trip is considered to start when engine gets enabled (e.g. LowVoltageSystemState in ON or START mode).
+           A trip is considered to end when engine is no longer enabled.
+           The signal may however keep the value of the last trip until a new trip is started.
+
+StartTime:
+  datatype: string
+  type: sensor
+  description: Start time of current or latest trip, formatted according to ISO 8601 with UTC time zone.
+  comment: This signal is supposed to be set whenever a new trip starts.
+           A new trip is considered to start when engine gets enabled (e.g. LowVoltageSystemState in ON or START mode).
+           A trip is considered to end when engine is no longer enabled.
+
+EndTime:
+  datatype: string
+  type: sensor
```

Comment:
Meeting notes:

- Daniel: Is rather an attribute, change only once per ignition cycle. 
- Sebastian: but could be not that intuitative to subscribe to an attribute to be informed when trip finishes.
- Daniel: Can be better to have both start time (attribute) and duration (sensor). 
- Decision: Erik to adapt to PR to use "both start time (attribute) and duration (sensor)". After update, others shall review. Merge or continued discussion on next meeting